### ISSUE TITLE
🍎 O-inari Origami ↔ v23.4.2

### ISSUE THREAD

**Issue by g0m** [itchio/itch-compatibility-watchlist#1236]
:rotating_light: game [O-inari Origami](https://sunspider.itch.io/origami) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/c6fd9662d40bb81ccf2cdfa1ecc47375).

:running: I think it might be a java for mac issue?

**Comment by fasterthanlime** [comment]
> I think it might be a java for mac issue?

Yep, I'd bet that the correct "natives" (.jnilib files) are not included for macOS - it's probably best to leave a comment to the developer on the game page, and link to this report.
